js/app/test/blocks_chained_events.spec.ts

```diff
@@ -19,10 +17,14 @@ test(".success event runs after function successfully completes. .success should
 	await page.click("text=Trigger Failure");
 	await last_iteration;
 	expect(textbox).toHaveValue("");
+});
 
+test(".success event runs after function successfully completes", async ({
```

Comment:
I think the issue was that there are two events being triggered here and we need to await the last one and the previous code wasn't doing that. I think we can just await the expected value in the textbox. Playwright will timeout if the text never shows up.

Reply:
Yeah that sounds simpler.